buildsystem/modules/FindPython.cmake

```diff
@@ -170,19 +170,24 @@ endforeach()
 # test all the found interpreters; break on success.
 foreach(PYTHON ${PYTHON_INTERPRETERS})
 
-	# ask the interpreter for the essential extension-building flags
-	py_get_config_var(INCLUDEPY PYTHON_INCLUDE_DIR)
-	py_get_config_var(LIBDIR PYTHON_LIBRARY_DIR)
-	py_get_lib_name(PYTHON_LIBRARY_NAME)
+	if (PYTHON STREQUAL PYTHON_EXECUTABLE)
+		find_package(PythonLibs REQUIRED)
```

Comment:
From `find_package(PythonInterp <other arguments>)` around line 73. Afterwards from the cache.
I'll have to tweak some pieces of this file to make the buildsystem happy on all platforms. You can skip reviewing this file for now.